test/test_win32_service.rb

```diff
@@ -10,14 +10,18 @@
 
 class TC_Win32_Service < Test::Unit::TestCase
   def self.startup
+    @@win_ver = Win32::Service.windows_version
     @@host = Socket.gethostname
-    @@service_name = 'stisvc'
-    @@elevated = Win32::Security.elevated_security?
+    @@service_name = @@win_ver < 6 ? 'Schedule' : 'LanmanServer'
+    # win32-security 0.3.1 crashes on 2003, so just assume elevated there
+    @@elevated = false
```

Comment:
Ah, crud, I removed Win XP support in win32-security 0.3.0. I didn't realize I was affecting Win2k3. Here was the original code:

https://github.com/djberg96/win32-security/blob/win32-security-0.2.5/lib/win32/security.rb